### PR TITLE
add babel transforms

### LINKED ISSUE #43: Crashes UglifyJS
We're using the latest package in a project and during the uglify build step, UglifyJS crashes because of index.js. Seems that built index.js still has `let` instead of `var`.

Specifically this line in `index.js` is causing the error:
```
    let tokens = state.tokens;
```

### PR DESCRIPTION
**Pull request by zentuit**
Fixes #43.

Adds babel transpiling to the browser build process.
I changed the 'main' script in package.json to point to the transpiled `markdown-it-attrs.browser.js` so during my build, the transpiled code gets included rather than the original `index.js`.

Also, I've upgraded to npm 5, so I have a `package-lock.json` -- if you want me to remove it, I can.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "markdown-it-attrs",
   "version": "1.0.0",
   "description": "Add classes, identifiers and attributes to your markdown with {} curly brackets, similar to pandoc's header attributes",
-  "main": "index.js",
+  "main": "markdown-it-attrs.browser.js",
```
Comment (arve0) [inline package.json]: This will break `require` in node.js.

File: package.json [hunk 2]
```diff
@@ -48,5 +52,17 @@
   "peerDependencies": {
     "markdown-it": ">=7.0.1"
   },
-  "tonicExampleFilename": "demo.js"
+  "tonicExampleFilename": "demo.js",
+  "browserify": {
```
Comment (arve0) [inline package.json]: Is this needed with `.babelrc`?

File: package.json (hunk 1 above)
Comment (zentuit) [inline package.json]: I don't believe it does.

I created a small project that had just `index.js` containing this:
```
var MdIt = require('markdown-it')
var mdAttrs = require('markdown-it-attrs')

var md = MdIt()

md.use(mdAttrs)

var text = 'test __some__ *markdown*{data-toggle=test} with attrs'

var output = md.render(text)
console.log('rendered text = ', output)
```
I got the same output when I ran the packaged v1.0.0 and my updated version.  Both produced the following with no errors:
```
rendered text =  <p>test <strong>some</strong> <em data-toggle="test">markdown</em> with attrs</p>
```
I did `node ./index.js` for both node v6.9.1 and node v8.2.1

File: package.json (hunk 2 above)
Comment (zentuit) [inline package.json]: You're right, its not needed since I added the transform param directly to browser script.  I'll remove.

**Comment by arve0** [comment]
Fixed in #45

**Pull request closed without merging**